### DISCUSSION TITLE
Is 1.10.0 considered stable?

### DISCUSSION THREAD

**Discussion by der-eismann** [coredns/coredns#5851] [Q&A] [answered]
I'm a bit confused because although it's been quite a while since 1.10.0 has been released there's still no entry in the release blog on the website and it mentions 1.9.3 as the latest release. Should we use it or not?

**Comment by chrisohaver** [comment, marked as answer]
1.10.0 is a normal stable release.  We had some maintainer turnover last year, and some hiccups in transitioning admin rights to maintain the website - leaving the website very outdated.

**Reply by der-eismann** [reply]
Thanks for the quick reply! In this case it would be great if the GitHub releases would contain more information about what has actually changed :slightly_smiling_face:

**Reply by chrisohaver** [reply]
There are release notes in the repo ... https://github.com/coredns/coredns/blob/master/notes/coredns-1.10.0.md

**Reply by der-eismann** [reply]
Then why not link them in the [release](https://github.com/coredns/coredns/releases/tag/v1.10.0) so people can find it?

**Reply by chrisohaver** [reply]
done
